app/connector/twitter/pom.xml

```diff
@@ -48,7 +67,16 @@
           <groupId>org.apache.camel</groupId>
           <artifactId>camel-core</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>org.twitter4j</groupId>
+            <artifactId>twitter4j-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <optional>true</optional>
```

Comment:
```suggestion
```

Not needed I think with the `<scope>provided</scope>`

Reply:
Fixed, thanks.